.github/workflows/build.yml

```diff
@@ -670,9 +670,9 @@ jobs:
           python-version:
             - 3.9
           package:
-            - "sqlalchemy>=2"
             - "sqlalchemy<2"
             - "numpy==1.19.5"
+            - "perspective-python<2"
```

Comment:
Did you mean to put <2 here or <3?

Reply:
good catch, copy-pasta from the sqlalchemy line